src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1786,26 +1788,28 @@ bool processOnChangeCondition
         if (condValues != NULL) {
             /* Check if some of the attributes in the NotifyCondition values list are in the entity.
              * Note that in this case we do a query for all the attributes, not restricted to attrV */
-            ContextElementResponseVector rawCerV;
+            ContextElementResponseVector rawCerV2;
```

Comment:
I added that call to release :-)
It *can* be used. It is just not a very good idea to shadow your own variables.

If you really want to keep using that name, why redeclare it?
Let's just reuse the same variable ...